protos/flow.proto

```diff
@@ -67,11 +67,12 @@ message RenameTableOption {
 }
 
 message RenameTablesInput {
+  reserved 2,4,5;
   string flow_job_name = 1;
   repeated RenameTableOption rename_table_options = 3;
-  optional string soft_delete_col_name = 4;
-  optional string synced_at_col_name = 5;
   string peer_name = 6;
+  string soft_delete_col_name = 7;
```

Comment:
this will break things

Reply:
this message is only used during resync and isn't persisted